### ISSUE TITLE
Enhance readme.md

### ISSUE THREAD

**Issue by kalsifer-742** [Kalsifer-7/MusicReactiveLedStrip#1]
- Enhance the readme and provide a little of documentation.
- Translate comments in english.

**Comment by aditya28sarin** [comment]
can you assign me this issue?

**Comment by kalsifer-742** [comment]
Yes of course. Thank you for interesting in such a little repo. I assigned the issue to you right away.
